src/Tools/PrepareTests/MinimizeUtil.cs

```diff
@@ -144,30 +146,39 @@ void writeHydrateFile()
             }
 
             var builder = new StringBuilder();
+            var fileName = isUnix ? "rehydrate.sh" : "rehydrate.cmd";
             foreach (var group in grouping)
             {
-                string filename;
                 builder.Clear();
-                if (isUnix)
+
+                if (group.Any(g => g.Count > 1))
```

Comment:
previously if there were no duplicates no rehydrate script would be written.  however all of the followup steps that use it assume its presence.  So we just write out a no-op script if there were no dupes.